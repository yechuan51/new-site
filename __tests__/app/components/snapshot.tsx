import { render } from "@testing-library/react";
import NavBarComp from "@/components/main_nav_bar/navBar";

it("render unchanged", () => {
  const { container } = render(
    <NavBarComp
      items={[
        { key: "item1", displayText: "Item 1", href: "/item1" },
        { key: "item2", displayText: "Item 2", href: "/item2" },
      ]}
    />
  );
  expect(container).toMatchSnapshot();
});