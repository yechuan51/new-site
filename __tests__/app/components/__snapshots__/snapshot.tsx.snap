// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render unchanged 1`] = `
<div>
  <nav
    class="container"
  >
    <p
      class="title"
    >
      <a
        href="/"
      >
        My website
      </a>
    </p>
    <ul
      class="navItemsList"
    >
      <li>
        <a
          href="/item1"
        >
          Item 1
        </a>
      </li>
      <li>
        <a
          href="/item2"
        >
          Item 2
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
